session_items.py

```diff
@@ -67,3 +67,7 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+def remove_item(id):
+    session['items'] = [item for item in get_items() if not (item['id'] == int(id))] 
```

Comment:
It's arguably simpler to write `item['id'] != int(id)` rather than `not (item['id'] == int(id))`